tracking.go

```diff
@@ -108,9 +121,14 @@ func (p *recvdPackets) prepareAckRange(protected bool, allowAckOnly bool) ackRan
 		if ok && !pk.acked2 {
 			if protected || !pk.protected {
 				needs_ack = true
+				newMinNotAcked2 = pn
 			}
 		}
 
+		if ok && pk.acked2 {
+			delete(p.packets, pn)
```

Comment:
I'm deleting this because I see no more use for use for storing this data. @ekr, can you confirm that this is the case, or is there something I am forgetting?